src/js/components/Tabs/Tabs.js

```diff
@@ -19,6 +30,7 @@ const Tabs = forwardRef(
       justify = 'center',
       messages,
       responsive = true,
+      step = 1,
```

Comment:
What are we recommending the step be for HPE Design System? Would it make sense to align the default value to that as opposed to requiring teams to add the prop each time?

Reply:
Or, set the default in the theme?